subt/main.py

```diff
@@ -931,6 +963,12 @@ def main():
         else:
             cfg['robot']['modules']['app']['init']['right_wall'] = args.side == 'right'
         cfg['robot']['modules']['app']['init']['timeout'] = args.timeout
+        if args.init_offset is not None:
+            x, y, z = [float(x) for x in args.init_offset.split(',')]
+            cfg['robot']['modules']['app']['init']['init_offset'] = [int(x*1000), int(y*1000), int(z*1000)]
+        if args.init_path is not None:
+            cfg['robot']['modules']['app']['init']['init_path'] = args.init_path
+
```

Comment:
Any specific reason why init offset is parsed here in main while init path is parsed in SubTChallenge init?

Reply:
well, I was not sure of the form :(. Moreover it is string while the list of tupples would be harder to store ... and floats. I expect revision "soon"